FirebaseCore/CHANGELOG.md

```diff
@@ -3,6 +3,18 @@
 - [added] The zip and Carthage distibutions now include the Swift extension frameworks. (#7819)
 - [changed] **Breaking change**: Update the minimum supported versions for the zip and Carthage
   distributions to iOS 11.0, tvOS 11.0 and macOS 10.13. (#9633)
+- [changed] **Breaking change**: CocoaPods Podfiles must include `use_frameworks!` or
+  `use_frameworks! :linkage => :static`.
+- [deprecated] Usage of the Firebase pod, the Firebase module (`import Firebase`), and `Firebase.h`
+  is deprecated. Use the specific Firebase product instead like: `pod 'FirebaseMessaging'` and
+  `import FirebaseMessaging`.
+- [changed] For Swift Package Manager installations, `import Firebase` will no longer implicitly
+  import Firebase Storage and Firebase Functions APIs. Use `import FirebaseStorage` and
+  `import FirebaseFunctions`.
+- [changed] Objective C++ clients should use `#import <FirebaseFunctions/FirebaseFunctions-Swift.h>`
+  and `#import <FirebaseStorage/FirebaseStorage-Swift.h>` to access Functions and Storage APIs,
+  respectively. Objective C++ clients using Swift Package Manager should find alternative
```

Comment:
I think you can just say `C++` here, since that should encompass both ObjC++ and C++.

Reply:
Did both to avoid confusion.